CICD troubleshooting dotUmbraco-file (#6800)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md

## Deployment failed

### File missing: The .umbraco file cannot be found in the root of the repository

The `.umbraco` file is missing or has been renamed. This file needs to be present in the root of the zipped package.

### File format Error: The .umbraco file is not valid

> The `.umbraco` file has invalid characters. Sometimes people need to change the folder structure of the repository and change the name of the default project. 
> Take care that the base field is not using backslashes ('\') as folder deminitor.

Reviewer: Suggestions to update the grammar and wording here a bit.
What do you think?

```suggestion
The `.umbraco` file has invalid characters. Sometimes people need to change the repository's folder structure and the default project's name. 
Ensure that the base field does not use backslashes ('\') as the folder denominator.
```

Author: much better - thank you 👍
